packages/drivers/routerlicious-driver/src/documentServiceFactory.ts

```diff
@@ -90,15 +90,39 @@ export class RouterliciousDocumentServiceFactory implements IDocumentServiceFact
             this.driverPolicies.enableRestLess,
             resolvedUrl.endpoints.ordererUrl,
         );
+
         // the backend responds with the actual document ID associated with the new container.
-        const documentId = await ordererRestWrapper.post<string>(
+        const res = await ordererRestWrapper.post<{ id: string, token?: string } | string>(
             `/documents/${tenantId}`,
             {
                 summary: convertSummaryToCreateNewSummary(appSummary),
                 sequenceNumber: documentAttributes.sequenceNumber,
                 values: quorumValues,
+                generateToken: true,
```

Comment:
I think the default here should be `this.tokenProvider.documentPostCreateCallback !== undefined`. Service generating a token takes a small amount of time, but it also increases the payload size which adds a little more time. For anyone not utilizing this feature, that additional latency is unnecessary.